### ISSUE TITLE
Segmentation when installed with noice.nvim

### ISSUE THREAD

**Issue by WilliamHsieh** [vimpostor/vim-tpipeline#40]
### Describe the bug

Apologize if this isn't related to this plugin. When this plugin is installed with [noice.nvim](https://github.com/folke/noice.nvim/) with `set laststatus=2`, moving cursor around is causing segmentation fault.

### To reproduce

1. nvim -nu /tmp/min.lua
2. try to move around (gg, G, Ctrl-D, Ctrl-U)
3. segmentation fault

/tmp/min.lua:
```lua
-- settings
vim.o.number = true
vim.o.relativenumber = true
vim.o.tabstop = 2
vim.o.shiftwidth = 2
vim.o.expandtab = true

-- problem causing setting
vim.o.laststatus = 0

-- plugins
vim.cmd([[set runtimepath=$VIMRUNTIME]])
vim.cmd([[set packpath=/tmp/nvim/site]])
local package_root = "/tmp/nvim/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
if vim.fn.isdirectory(install_path) == 0 then
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
require("packer").startup {
  {
    "wbthomason/packer.nvim",
    "vimpostor/vim-tpipeline",
    {
      "folke/noice.nvim",
      requires = {
        "MunifTanjim/nui.nvim",
        "rcarriga/nvim-notify",
      },

      config = function()
        require("noice").setup()
      end,

    },
  },
  config = {
    package_root = package_root,
    compile_path = install_path .. "/plugin/packer_compiled.lua",
  },
}
require("packer").sync()
```

### Terminal emulator

Alacritty

### Debug output

```JSON
can't show the output because of segmentation fault
```

**Comment by vimpostor** [comment]
Can't reproduce with your minimal config.
- In your minimal config you set `laststatus=0` but in your description you talk about `laststatus=2`, which one of the both is it now?
- What version of neovim are you using?
- Note that `laststatus=2` with this plugin will not make not much sense anyway, because you will end up with duplicated statuslines

**Comment by WilliamHsieh** [comment]
> * In your minimal config you set `laststatus=0` but in your description you talk about `laststatus=2`, which one of the both is it now?

It's a typo in the description, I mean laststatus=0, sorry.

> * What version of neovim are you using?

0.8.0

This is weird. Segmentation fault only occurs on macOS and Linux, but on Windows wsl everything is fine.
After I upgrade to `nvim 0.8.1`, everything works out well, so I guess it's an upstream bug?

I'm closing this for now.
I have another issue with noice, and I will post another issue. Thank's.

**Comment by vimpostor** [comment]
>After I upgrade to `nvim 0.8.1`, everything works out well, so I guess 
>it's an upstream bug?

Yes, in general if it segfaults, it's very likely an upstream bug.
Good to hear that it is resolved with neovim 0.8.1
